### DISCUSSION TITLE
Linux Server

### DISCUSSION THREAD

**Discussion by GeekPR1977** [apohl79/audiogridder#1375] [Ideas]
Wish this could be installed into a Linux server, or as a Docker! would be so cool!

**Comment by Serabass** [comment]
I want this feature.
I have a k3s cluster with 15 nodes and I would like to deploy AG with N replicas and use load balanced VST-server when I work on DAW in Windows.

**Comment by kcoul** [comment]
I'm very close to getting this working, there are just a few linker errors to work through left to go.
